designs/administrator/olm/installing-operator-long-running-component/index.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Operator Installing with Common Long Running Styling
+
+Shortly after the new screen to [Convey an operator is installing](http://openshift.github.io/openshift-origin-design/designs/administrator/olm/convey-installing-operator/) was developed, it was realized that other actions in the console could make use of a similar interaction. This design conveys a more generic version of the operator installing screen, and examples of perhaps how other actions could use a similar interaction and styling.
+
+## Operator Installation
+
+These screens show how the existing Installing Operator feature could appear using newer, more generic, visuals, which could perhaps be reused for other use cases.
+
+### Installing an operator
+![Install operator](img/longRun-1-1-installing.png)
```

Comment:
Is the text recommendation finalized here? InstallSucceeded is in camel case and next to it it says "Installing Container Security" and "waiting for install components to report healthy", so has the install succeeded? I think at the least install succeeded shouldn't be camel case if we can work around that.

Reply:
Unfortunately for now this text is just coming straight from the backend, but perhaps in the future we can display our own strings based on status of the resources.